zerver/lib/notification_data.py

```diff
@@ -186,6 +205,8 @@ def get_email_notification_trigger(self, acting_user_id: int, idle: bool) -> Opt
             return NotificationTriggers.PRIVATE_MESSAGE
         elif self.mention_email_notify:
             return NotificationTriggers.MENTION
+        elif self.followed_topic_wildcard_mention_email_notify:
+            return NotificationTriggers.FOLLOWED_TOPIC_WILDCARD_MENTION
         elif self.wildcard_mention_email_notify:
             return NotificationTriggers.WILDCARD_MENTION
```

Comment:
This is an interesting corner case about precedence that we should discuss in #design. (It's easy to change, so not a blocker for this PR). I think it's possible that in the case where `wildcard_mention_email_notify` was already true, even though the "followed topic" case is more fine-grained, it may not be worth calling out that you follow the topic in the description of why you got the email, since you are getting it because of the wildcard mention regardless in your configuration?